webpack/ForemanTasks/Components/TaskDetails/TaskDetailsSelectors.js

```diff
@@ -29,7 +29,9 @@ export const selectErrors = state => {
 };
 
 export const selectProgress = state =>
-  selectTaskDetails(state).progress * 100 || 0;
+  selectTaskDetails(state).progress
+    ? selectTaskDetails(state).progress.toFixed(4) * 100
+    : selectTaskDetails(state).progress * 100 || 0;
```

Comment:
This trenary branch will only be reached if    `selectTaskDetails(state).progress` is undefined or 0 - so we can fall back to 0 directly
```suggestion
    : 0;
```